<?xml version="1.0"?>

<launch>

  <!--settings for nav-->
  <arg name="point_seq" default="[1, 3, 0, 4.5, 3, 0, 4.5, -1, 0, 0, 0, 0]" doc="points sequence for nav plan. format: x1,y1,z1,x2,y2,z2,..."/>
  <arg name="yaw_seq" default="[0, 270, 180, 0]" doc="angles sequence that coresponds point_seq. format: yaw1,yaw2,... "/>
  <arg name="gazebo" default="false" doc="execute armadillo2 inside gazebo sim"/>

  <node name="move_base_client" pkg="elevator" type="nav_client.py" output="screen">
    <rosparam param="point_seq" subst_value="True">$(arg point_seq)</rosparam>
    <rosparam param="yaw_seq" subst_value="True">$(arg yaw_seq)</rosparam>
  </node>

</launch>
